chest.lua

```diff
@@ -11,35 +11,69 @@ local function get_item_count(pos, player, count)
 	return count
 end
 
+-- Just in case we are under MineClone
+local default = rawget(_G, "default") or nil
+if default == nil then
+	default = rawget(_G, "mcl_sounds")
```

Comment:
Please add local variables to specify `default.node_sound_wood_defaults` game-dependent configuration. This line relies on 100% API of `default` and `mcl_sounds`, which is by far not true.

Suggestion: use something similar to pipeworks: https://github.com/mt-mods/pipeworks/blob/master/crafts.lua#L1

Reply:
I don't quite understand what your pointing out with this.

The code makes it so if default doesn't exist our usage of `default.node_sound_wood_defaults()` is the exact same, with 0 difference code wise. (Unless we use other things in default this works just fine, but I guess I could take the output from it as a variable and change based on availability)

Reply:
Like [https://github.com/Beanzilla/item_replicator/blob/master/register.lua#L314-L324](https://github.com/Beanzilla/item_replicator/blob/master/register.lua#L314-L324)?

Reply:
Yes, the code stays the same, but `default` it not guaranteed to support all MTG API that's expected from it (if using MineClone). Even though it does not result in errors now, it's a hack that is better implemented in the manner of the mod you linked above.

Reply:
Yeah I've reworked this to like my mod, we use the one feature we need from it and not try to load it in it's entirety.